lib/pg_party/model_decorator.rb

```diff
@@ -60,21 +60,21 @@ def list_partition_key_in(*values)
       end
     end
 
-    alias_method :hash_partition_key_in, :list_partition_key_in
+    alias hash_partition_key_in list_partition_key_in
```

Comment:
Not sure how I feel about this one...

Reply:
Reverted back `alias_method` and updated Rubocop configuration to make `alias_method` preferred over `alias`.